tools/postgres/ssl/pg_hba.conf

```diff
@@ -1,2 +1,3 @@
 # TYPE  DATABASE        USER            ADDRESS                 METHOD
+local	all				all										trust
```

Comment:
Why do we need this line ?

Documentation says `This record matches connection attempts using Unix-domain sockets. Without a record of this type, Unix-domain socket connections are disallowed.`

Reply:
It's for initial script that is applied after docker compose is up